CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ incremented for features.
 * cli: Add version number from programs `Cargo.toml` into extracted IDL ([#1061](https://github.com/project-serum/anchor/pull/1061)).
 * lang: Add `deprecated` attribute to `Loader`([#1078](https://github.com/project-serum/anchor/pull/1078)).
 * lang: the `init_if_needed` attribute now checks that given attributes (e.g. space, owner, token::authority etc.) are validated even when init is not needed ([#1096](https://github.com/project-serum/anchor/pull/1096)).
+* lang: Change commitment message recent to processed ([#1128](https://github.com/project-serum/anchor/pull/1128))
```

Comment:
this msg is in the wrong place. `0.19.0` is released already. you can add this to a new `fixes` section in the `Unreleased` section